lib/vintage_net_mobile/modem/huawei_E3372.ex

```diff
@@ -0,0 +1,88 @@
+defmodule VintageNetMobile.Modem.HuaweiE3372 do
+  @behaviour VintageNetMobile.Modem
+
+  alias VintageNet.Interface.RawConfig
+  alias VintageNetMobile.{CellMonitor, Chatscript, ExChat, ModemInfo, PPPDConfig, SignalMonitor}
+  alias VintageNetMobile.Modem.Utils
+
+
+  @moduledoc """
+  Huawei E3372 support
+
+  This modem needs a usb modeswitch, the follwing can serve as an example
+
+  ```elixir
+  def modemswicth(product_id \\ "14fe") do
+      Toolshed.cmd("usb_modeswitch -v 12d1 -p #{product_id} -X")
```

Comment:
Use `System.cmd("usb_modeswitch", ["-v", "12d1", "-p", product_id, "-X"])` here to avoid bringing in `Toolshed`.

Reply:
Do we want the polling to be part of `vintage_net_mobile`, then I would be happy t ocontinue this

Reply:
So long as it's a relatively simple piece of code (which it looks), then I'd prefer that it be in `vintage_net_mobile` so that as modems are added, it can be refactored to support them as well.

Reply:
Currently, I have something that will start two usb "detectors". One that does a modem switch and one that does the actual connect. But I am not sure where I should supervisor them and when in this package?